src/main/java/com/jlr/ttl/poc/loadmanager/models/Vehicle.java

```diff
@@ -2,28 +2,38 @@
 
 import jakarta.persistence.*;
 
+import java.util.Date;
+
 @Entity
 @Table(name = "vehicles")
 public class Vehicle {
+
     @Id
-    @Column(name="id")
-    private Integer vin;
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column
+    private Integer id;
+
+    @Column(name = "short_vin")
+    private String vin;
     @Column(name = "loc_code")
     private String loc;
 
-    public Vehicle(Integer vin, String loc_code) {
+    @Column(name = "end_date")
+    private Date date;
+
+    public Vehicle(String vin, String loc_code) {
         this.vin = vin;
         this.loc = loc_code;
     }
 
     public Vehicle() {
     }
 
-    public Vehicle(Integer vin) {
+    public Vehicle(String vin) {
```

Comment:
Remove constructors and getters and setters. Use the following annotations instead:

AllArgsConstructor
NoArgsConstrutctor
Data
Build